src/Sylius/Component/Report/Renderer/RendererInterface.php

```diff
@@ -18,9 +18,9 @@
 {
     /**
      * @param ReportInterface $report
-     * @param Data $data
+     * @param Data            $data
      *
-     * @return string
+     * @return mixed
```

Comment:
Though the interface said `string`, the implementations were returning `Response`. Of course now that's no longer the case, but it got me thinking - should it always return a string?